packages/client/wallets/smart-wallet/src/api/APIErrorService.ts

```diff
@@ -0,0 +1,63 @@
+import {
+    CrossmintServiceError,
+    JWTDecryptionError,
+    JWTExpiredError,
+    JWTIdentifierError,
+    JWTInvalidError,
+    type SmartWalletSDKError,
+} from "@/types/Error";
+
+export type CrossmintAPIErrorCodes =
+    | "ERROR_JWT_INVALID"
+    | "ERROR_JWT_DECRYPTION"
+    | "ERROR_JWT_IDENTIFIER"
+    | "ERROR_JWT_EXPIRED";
+
+export class APIErrorService {
+    constructor(
+        private errors: Partial<
+            Record<
+                CrossmintAPIErrorCodes,
+                // biome-ignore lint/suspicious/noExplicitAny: we should be able to use 'any' when ingesting JSON from external sources
```

Comment:
We don't have biome setup yet in this repo, so these must be the default rules when you have it installed.